packages/loader/container-loader/src/container.ts

```diff
@@ -1992,6 +1993,7 @@ export class Container
 		batch?: boolean,
```

Comment:
It feels like submitSummaryMessage() should be more explicit and pass ref seq number to submitMessage().
We do not have any better info (I thought seq number would be in ISummaryContent, but there is nothing there).
So maybe add optional argument to submitSummaryMessage() API such that runtime (in future) could supply ref sequence number, and thus (for example) produce a summary (from old state) while inbounding ops. Today that's not possible as our DDSs do not have capability to summaize old state (and exclude local changes), but maybe in future we will have that capability, and summarizer code will need to supply ref seq number.